packages/less/Gruntfile.js

```diff
@@ -223,12 +216,7 @@ module.exports = function(grunt) {
                 command: "node build/rollup.js --browser --out=./tmp/browser/less.min.js"
             },
             test: {
-                command: [
```

Comment:
This is the root cause for the CI suspension, we don't need these code anymore since we upgrade ts-node.